internal/cmd/skupper/token/kube/token_issue.go

```diff
@@ -173,6 +173,11 @@ func (cmd *CmdTokenIssue) WaitUntil() error {
 		}
 
 		if token != nil && token.IsReady() {
+			token.TypeMeta = metav1.TypeMeta{
+				APIVersion: "skupper.io/v1alpha1",
+				Kind:       "AccessGrant",
```

Comment:
Should it be AccessToken?

Reply:
I think that makes sense.

Reply:
@lynnemorrison the code in token_redeem is reading the file as an AccessGrant to covert it later as an AccessToken (https://github.com/skupperproject/skupper/blob/46036fcdb2f8d34a4d45f867a56286d453d1f89a/internal/cmd/skupper/token/kube/token_redeem.go#L123)...

Will change that in both places.